scripts/assemble_figure.py

```
@@ -0,0 +1,478 @@
+    consistent).
+    """
+    if "/" in value or value.lower().endswith((".svg", ".png", ".emf")):
+        return None
+    json_index = library_root / "library" / "index.json"
+    if json_index.exists():
+        try:
+            for r in json.loads(json_index.read_text()):
+                if r.get("id") == value and r.get("attribution_required"):
+                    return r
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)